### DISCUSSION TITLE
Block users and list of blocked users endpoint

### DISCUSSION THREAD

**Discussion by kevincador** [trakt/trakt-api#336] [Q&A] [answered]
Hi,

I saw the website has been updated to be able to block users/members via comments. Would it be possible to have some endpoints to be able to handle that from 3rd party apps?

Thanks!

**Comment by rectifyer** [comment, marked as answer]
The APIs exist for this.

`last_activities` includes **`comments.blocked_at`** to know when you need to re-sync the list of blocked members.
https://trakt.docs.apiary.io/#reference/sync/last-activities/get-last-activity

`/users/hidden/comments`
https://trakt.docs.apiary.io/#reference/users/hidden-items/get-hidden-items

Block a comment user:
https://trakt.docs.apiary.io/#reference/users/add-hidden-items/add-hidden-items

Unblock a comment user:
https://trakt.docs.apiary.io/#reference/users/remove-hidden-items/remove-hidden-items

**Reply by kevincador** [reply]
Okay my bad. I always thought the hidden-items was about the comment, not the user but I now see that the section is comment and the object is the user 🤦‍♂️
I'll make the change to take this into account. Thanks!
